src/Calendars/ChineseCalendar.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Spatie\Holidays\Calendars;
+
+use Carbon\CarbonImmutable;
+use DateTimeZone;
+use IntlDateFormatter;
+
+trait ChineseCalendar
```

Comment:
nên đổi Chinese thành Lunar

Reply:
Good point, I did some research, Chinese Calendar could be a more precise name.